utils/rpms/daos.spec

```
@@ -9,9 +8,13 @@
 %global mercury_version 2.0.0~rc1-1%{?dist}
 %endif
 
+%if %{undefined scons_args}
+%define scons_args BUILD_TYPE=release
+%endif
```

The _idiomatic way to do this is:

```suggestion
%{!?scons_args:%global BUILD_TYPE release} 
```

However more importantly, this defeats our "secure by default" stance.  IOW, this should be the case without even having to use `BUILD_TYPE=release` and then we should have a test that confirms it.

---

If scons_args is not defined then scons will build with BUILD_TYPE default defined in SContruct

scons %{?_smp_mflags}      \
      --config=force       \
      --no-rpath           \
      USE_INSTALLED=all    \
      CONF_DIR=%{conf_dir} \
      PREFIX=%{?buildroot} \
%if %{defined scons_args}
     %{scons_args}
%endif

---

> If scons_args is not defined then scons will build with BUILD_TYPE default defined in SContruct
> 
> scons %{?_smp_mflags}
> --config=force
> --no-rpath
> USE_INSTALLED=all
> CONF_DIR=%{conf_dir}
> PREFIX=%{?buildroot}
> %if %{defined scons_args}
> %{scons_args}
> %endif

The idiomatic way is to use `%{?scons_args}` rather than wrapping it with `%if %{defined ...}`.